Source/WebKit/UIProcess/cairo/BackingStoreCairo.cpp

```diff
@@ -101,8 +101,8 @@ void BackingStore::incorporateUpdate(UpdateInfo&& updateInfo)
         return;
 
 #if ASSERT_ENABLED
-    IntSize updateSize = updateInfo.updateRectBounds.size();
-    updateSize.scale(m_deviceScaleFactor);
+    IntSize updateSize = { static_cast<int>(ceilf(updateInfo.updateRectBounds.size().width() * m_deviceScaleFactor)),
```

Comment:
I think this can be `IntSize updateSize = expandedIntSize(updateInfo.updateRectBounds.size() * m_deviceScaleFactor);`

Reply:
That seems smart!
I will change the codes.